packages/pigment-css-unplugin/src/index.ts

```diff
@@ -326,14 +324,20 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
           };
         }
 
+        const rootPath = process.cwd();
         const slug = slugify(cssText);
-        const cssFilename = `${slug}.pigment.css`;
-        const cssId = `./${cssFilename}`;
-        cssFileLookup.set(cssId, cssFilename);
-        cssLookup.set(cssFilename, cssText);
+        const cssFilename = path
+          .normalize(`${id.replace(/\.[jt]sx?$/, '')}-${slug}.pigment.css`)
+          .replace(/\\/g, path.posix.sep);
+
+        const cssRelativePath = path.relative(rootPath, cssFilename).replace(/\\/g, path.posix.sep);
+        const cssId = `\0${cssRelativePath}`;
```

Comment:
Why the `\0`? a comment would help a lot.

Reply:
Added